plugins/json/tests/json_04_arrays_of_multi_datatypes.c

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+ * Copyright (C) 2023 Commissariat a l'energie atomique et aux energies alternatives (CEA)
+ * All rights reserved.
+ *
```

Comment:
You added some change in the test file compare.
```suggestion
/*******************************************************************************
 * Copyright (C) 2023-2024 Commissariat a l'energie atomique et aux energies alternatives (CEA)
 * All rights reserved.
 *
```

Reply:
you are right!